OpenRA.Mods.Common/Traits/Turreted.cs

```diff
@@ -117,7 +117,7 @@ void ITick.Tick(Actor self)
 		protected virtual void Tick(Actor self)
 		{
 			// NOTE: FaceTarget is called in AttackTurreted.CanAttack if the turret has a target.
-			if (attack != null)
+			if (IsTraitDisabled || attack != null)
 			{
 				// Only realign while not attacking anything
 				if (attack.IsAiming)
```

Comment:
This will throw a NRE if `attack` is null and the trait is disabled.

Reply:
fixed